v2/azure-arm.yaml

```diff
@@ -14,8 +14,8 @@ typesOutputPath: api
 typeRegistrationOutputFile: internal/controllers/controller_resources_gen.go
 
 # rootUrl is the root URL for ASOv2 repo
-rootUrl: https://github.com/Azure/azure-service-operator/tree/main/v2
-# samplesPath is the base Path used for walking through to samples when we generate a list of supported resources
+rootUrl: https://github.com/Azure/azure-service-operator/tree/main/v2/
+# samplesPath is the relative path from the current directory used for walking through to samples when we generate a list of supported resources
```

Comment:
Is this the current directory though? What if we ran `generator gen-types v2/azure-arm.yaml` ?

It should be relative to the yaml file.

Reply:
Done! Have updated the comment